node/src/reactor.rs

```diff
@@ -925,3 +929,32 @@ where
         .map(move |effect| wrap_effect(wrap.clone(), effect))
         .collect()
 }
+
+pub(crate) fn handle_fetch_response<R, T>(
+    reactor: &mut R,
+    effect_builder: EffectBuilder<<R as Reactor>::Event>,
+    rng: &mut NodeRng,
+    sender: NodeId,
+    serialized_item: Vec<u8>,
+) -> Effects<<R as Reactor>::Event>
+where
+    T: Item,
+    R: Reactor,
+    <R as Reactor>::Event: From<fetcher::Event<T>> + From<BlocklistAnnouncement<NodeId>>,
+{
+    match fetcher::Event::<T>::from_get_response_serialized_item(sender, &serialized_item) {
+        Some(fetcher_event) => {
+            Reactor::dispatch_event(reactor, effect_builder, rng, fetcher_event.into())
+        }
+        None => {
+            info!(
+                "{} sent us a {:?} item we couldn't parse! Banning",
```

Comment:
Just a minor nit, but this might be a good opportunity to make the ban-related messages consistent, even if just for the sake of easier log parsing.

Please consider picking one of:
* "Banning"
* "banning"
* "banning peer"

Reply:
Done in https://github.com/casper-network/casper-node/pull/2428/commits/fb0a0d55d0ae7af73c86fcc21bda9e37ec9aff38.